packages/react-reconciler/src/__tests__/ReactIncrementalErrorHandling-test.js

```diff
@@ -745,14 +745,16 @@ describe('ReactIncrementalErrorHandling', () => {
     expect(ReactNoop.flush).toWarnDev(
       'Warning: React.createElement: type is invalid -- expected a string',
     );
-    expect(ReactNoop.getChildren()).toEqual([
+    expect(ReactNoop.getChildren()).toErrorWithStack([
```

Comment:
Same as below. It was not throwing an error so it shouldn’t throw one now either.